aria/modeling/service_template.py

```diff
@@ -2131,13 +2132,15 @@ def resolve(self, model_storage):
         # moved to.
         plugins = model_storage.plugin.list()
         matching_plugins = []
-        for plugin in plugins:
-            # TODO: we need to use a version comparator
-            if (plugin.name == self.name) and \
-                    ((self.version is None) or (plugin.package_version >= self.version)):
-                matching_plugins.append(plugin)
+        if plugins:
+            for plugin in plugins:
+                if (plugin.name == self.name) and \
```

Comment:
I strongly disagree, especially in this case when there is `or` and `and` in the same clause. Do you otherwise expect the reader to know the priority of logical binary operations? Parentheses remove any potential confusion.